lib/renuo_cms_rails/configuration.rb

```diff
@@ -0,0 +1,29 @@
+module RenuoCmsRails
+  class << self
+    attr_writer :configuration
```

Comment:
Isn't the convention here to call this `config`? Here you're sticking to it: https://github.com/renuo/renuo-cms-rails/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R84